7_react/react2/week1/github_user_searcher/src/components/SearchInput.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from "react";
+import { GithubSearchContext } from "../context/GithubSearchContext";
+
+const SearchInput = () => {
+  const { query, setQuery, fetchUsers } = useContext(GithubSearchContext);
```

Comment:
the more common way how to access the context could be
```
// GithubSearchContext.jsx

export const useGithub = () => {
  return useContext(GithubSearchContext);
};
```
```
// Search Input.jsx
const { query, setQuery, fetchUsers } = useGithub()
```

But your way is also correct. Well done